explainer.md

```diff
@@ -17,86 +15,140 @@ The two drafts apply to both TLS server certificates, where the server authentic
+
+### Background
+
+TLS uses X.509 certificates to associate DNS names, or other application identifiers, with the authoritative server's TLS keys. These associations are signed by certificate authorities (CAs) and are presented by servers to clients. Each client curates a set of CAs, called *trust anchors*, whose associations the client accepts. If the client's trust anchors can be trusted to only issue correct associations, the client can rely on TLS to securely connect to servers.
+
+In this system, the client's trust anchors directly impact service availability and user security:
+
+* If the authoritative server is unable to obtain or present a certificate from a CA trusted by the client, it cannot serve that client. Without trustworthy CAs that servers can use, the client cannot connect.
+
+* If a client trusts an untrustworthy CA, that CA can forge an invalid association with an attacker's TLS key. The attacker can then intercept the client's connection to the server. Note the CA used by the authoritative server is irrelevant to this attack, only client trust. Conversely, until the client no longer trusts the untrustworthy CA, the attack has not been mitigated.
+
+This dynamic means PKIs necessarily evolve over time. To meet user security needs, clients remove compromised or untrustworthy CAs, so that they cannot be used to intercept TLS. To meet service availability needs, clients add trustworthy CAs that add net value to the ecosystem, so that servers are able to obtain acceptable certificates.
+
+Some transitions combine the two: a long-lived CA key is more likely to be compromised and presents a higher user security risk. To rotate a key in a PKI, the client would add a new key from the CA operator, and then remove the old one.
+
+### Client Diversity
 
-A single subscriber may need to interoperate with relying parties that trust different sets of CAs. TLS 1.3 defines the [`certificate_authorities` extension](https://www.rfc-editor.org/rfc/rfc8446#section-4.2.4) to accommodate this. It allows the subscriber to provision multiple certificates and select the one that will allow the relying party to accept its TLS key. This is analogous to parameter negotiation elsewhere in TLS.
+As with other TLS parameters, client populations do not have completely uniform trust anchors.
+
+PKI transitions described above inherently create this diversity, if for no other reason than out-of-date clients in the ecosystem. These clients could range from unupdatable TV set-top boxes to some IoT device to an individual user's browser that has not yet, or could not, communicate with its update service.
+
+Beyond temporal divergence, different clients have different needs and make different decisions on behalf of their users. For example, within the Web PKI, root programs make trust decisions independently. Some more specialized clients, such as mobile apps, may even pin to one or two specific CAs.
```

Comment:
We can avoid people rabbit-holing on pinning by saying something more generally true, such as:

"Some more specialized clients, such as mobile apps, may be constrained to trusting one or two specific CAs"